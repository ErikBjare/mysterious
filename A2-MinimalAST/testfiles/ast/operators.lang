int main() {
	int x;
	int y;
	x = 2-1;
	y = x*5%3+1;
}