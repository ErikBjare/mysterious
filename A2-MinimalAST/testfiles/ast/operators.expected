Program
  List
    Function
      IdDecl ID="main" TYPE="int"
      List
      Block
        List
          IdStmt
            IdDecl ID="x" TYPE="int"
            Opt
          IdStmt
            IdDecl ID="y" TYPE="int"
            Opt
          Assign
            IdUse ID="x"
            Sub
              Literal LITERAL="2"
              Literal LITERAL="1"
          Assign
            IdUse ID="y"
            Add
              Mul
                IdUse ID="x"
                Mod
                  Literal LITERAL="5"
                  Literal LITERAL="3"
              Literal LITERAL="1"
